main.py

```diff
@@ -1,7 +1,42 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Action",
+        "Dramma"
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. This is corrected later in the code, but it's better to avoid such typos initially.